driver_test.go

```diff
@@ -373,6 +373,36 @@ func Test_Ping_Error(t *testing.T) {
 		})
 }
 
+func Test_ErrSkip_Error(t *testing.T) {
+	t.Parallel()
+
+	var skipErr = driver.ErrSkip
+
+	oteltest.New(
+		oteltest.MockDatabase(func(m sqlmock.Sqlmock) {
+			m.ExpectPing().
+				WillReturnError(skipErr)
```

Comment:
Can we just use `driver.ErrSkip` directly?

Reply:
Done ✅